app/hooks/android_adjust_soft_keyboard/index.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import RNUtils from '@mattermost/rnutils';
+import {useEffect, useRef} from 'react';
+import {Keyboard} from 'react-native';
+import {Navigation} from 'react-native-navigation';
+
+import type {AvailableScreens} from '@typings/screens/navigation';
+
+export function useAndroidAdjustSoftKeyboard(screen?: AvailableScreens) {
```

Comment:
The name of the function talks about android, but I don't see any `if (Platform.os === 'android')`. Is that correct?

Reply:
True, but iOS causes a noop